SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialPackageMapClient.h

```diff
@@ -62,10 +62,9 @@ class SPATIALGDK_API USpatialPackageMapClient : public UPackageMapClient
 
 	virtual bool SerializeObject(FArchive& Ar, UClass* InClass, UObject*& Obj, FNetworkGUID *OutNetGUID = NULL) override;
 
-private:
-	UPROPERTY()
-	USpatialClassInfoManager* ClassInfoManager;
```

Comment:
Do we need the declaration of USpatialClassInfoManager "class USpatialClassInfoManager;" ? Also in the cpp do we still need the include of SpatialClassInfoManager.h?

Reply:
Ok we do need the header in the cpp. Let's just remove the declaration.

Reply:
I don't think there is a header for the SpatialClassInfoManager in the cpp?